docs/content/topics/scripting/guide.md

```diff
@@ -9,1401 +9,867 @@ aliases:
+Hello, Brigade!
+```
 
-Brigade is designed to make it easy for you to extract information from the event and project
-and sent it into a job. Here are two ways to share information with jobs:
+We can update the example to print the project ID as well.
 
 ```javascript
-const { events, Job } = require("brigadier")
-
-events.on("exec", (e, p) => {
-  var echo = new Job("echo", "alpine:3.4")
-  echo.tasks = [
-    "echo Project " + p.name,
-    "echo Event $EVENT_NAME"
-  ]
+const { events } = require("@brigadecore/brigadier");
 
-  echo.env = {
-    "EVENT_NAME": e.type
-  }
+events.on("brigade.sh/cli", "exec", async event => {
+  console.log("Project: " + event.project.id);
+  console.log("Hello, " + event.payload + "!");
+});
 
-  echo.run()
-})
+events.process();
 ```
-[brigade-14.js](../../examples/brigade-14.js)
-
-In the above code, we create a job named `echo` and we run two tasks. In the first, we
-directly inject the project name (`p.name`) into the task command before the task is run.
 
-In the second case, we use environment variables to pass a name/value pair into the
-command, and then it is evaluated at runtime.
+The following output is generated:
 
-`echo.env` is the place to set environment variables for the container. The variable set
-to `EVENT_NAME` there is accessible inside the pod as `$EVENT_NAME`.
+```console
+$ brig event create --project first-payload --payload "Brigade" --follow
 
-If we look at the output of the pod, we'll see this:
+Created event "b45720c4-115c-4c9b-b668-a872479f2210".
 
+Waiting for event's worker to be RUNNING...
+2021-09-23T16:30:29.267Z INFO: brigade-worker version: 927850b-dirty
+Project: first-payload
+Hello, Brigade!
 ```
-$ kubectl logs echo-1504074306432-master
-Project github.com/brigadecore/empty-testbed
-Event exec
-```
-
-At this point we have seen how we can access information about the project and event. In
-the next section we are going to turn to storage. We are going to see how Brigade provides
-ways for builds and jobs to share storage space.
-
-## Storing Data with Caches and Shared Space
-
-There are many ways that developers can store and retrieve data from Brigade. For example,
-object storage systems like Azure Object Storate or Amazon S3 or hosted database
-providers. Brigade developers may choose to use those tools from within jobs.
-
-But Brigade comes with a few built-in options that are useful in writing basic Brigade builds,
-and which don't require modifying your containers. In this part of the guide, we cover two
-built-in shared directories.
-
-### Build Storage (Shared Space)
 
-Each build gets its own shared storage. This is a file path that can be accessed by every
-job during the build, but which does not survive after the build has completed.
+Note that some event and project data should be treated with care. Things like
+`event.project.secrets` or event `event.worker.git.cloneURL` might not be the
```

Comment:
s/or event/or even/

Reply:
I'm not 100% clear on what the exposure is if a clone URL were leaked.

Reply:
Maybe if it were a private repo not intended to be known to others?

Reply:
Fair point. I think that's one of those edge cases where one might consider standing up their own brigade instance. No need to get into that here. Current level of details is good.